Libraries/Components/RefreshControl/RefreshControl.js

```diff
@@ -117,6 +117,11 @@ const RefreshControl = React.createClass({
      * @platform android
      */
     size: React.PropTypes.oneOf(RefreshLayoutConsts.SIZE.DEFAULT, RefreshLayoutConsts.SIZE.LARGE),
+    /**
+     * Progress view offset (display points)
```

Comment:
No need to mention display points here since pretty much all the APIs use that. I'd also add 'top offset' instead of 'offset'.